flashlight/fl/tensor/backend/jit/JitTensor.h

```diff
@@ -20,8 +20,8 @@ namespace fl {
 template <typename T>
 class JitTensor : public JitTensorBase {
  protected:
-  Tensor fromNode(Node* node) const override {
-    return toTensor<JitTensor>(node);
+  Tensor fromSharedNode(std::shared_ptr<ShareData> sharedNode) const override {
```

Comment:
To make sure I understand correctly — a shared ptr to `SharedData` serves only to pass refcount operations themselves on `Node`s around?

Reply:
> SharedData serves only to pass refcount operations themselves on Nodes around

Actually no. `RefCount` and `SharedData` solves orthogonal problems -- the former is for memory management of `Node`, the latter is for mutation under aliasing (alias from shallowCopy or indexing).

Note that we used `SharedData` to handle alias at many other places -- [Variable::SharedData](https://github.com/flashlight/flashlight/blob/main/flashlight/fl/autograd/Variable.h#L327-L328), [ArrayFireTensor](https://github.com/flashlight/flashlight/blob/main/flashlight/fl/tensor/backend/af/ArrayFireTensor.h#L30-L32), and [OneDnnTensor](https://github.com/flashlight/flashlight/blob/main/flashlight/fl/tensor/backend/onednn/OneDnnTensor.h#L60-L61).